src/features/user/api/fetch-profile-image.ts

```diff
@@ -0,0 +1,25 @@
+import { ref, listAll, getDownloadURL } from "firebase/storage";
+import { imageDatabase } from "../../../shared/config/firebase-image";
+
+export const fetchProfileImage = async (id: number) => {
+  try {
+    const res = await listAll(ref(imageDatabase, "uploads"));
+
+    const promises = res.items.map(async (item) => {
+      const url = await getDownloadURL(item);
+      const originalString = item.fullPath;
+      const trimmedString = parseInt(originalString.substring("uploads/".length));
+      if (trimmedString === id) {
```

Comment:
Czym się różni ten cały plik od fetch-user-image?

Reply:
Oo, wsm to niczym, ale już usnąłem jeden i używam tylko jednego